src/game/chars/CCharUse.cpp

```diff
@@ -1209,10 +1209,18 @@ bool CChar::FollowersUpdate( CChar * pChar, short iFollowerSlots, bool fCheckOnl
                 ++it;
             }
 
-            if (!fExists && ((short)(m_followers.size()) < iMaxFollower || IsPriv(PRIV_GM)))
+            if (!fExists && ((short)(m_followers.size()) + iFollowerSlots <= iMaxFollower || IsPriv(PRIV_GM)))
             {
                 if (!fCheckOnly)
-                    m_followers.emplace_back(pChar->GetUID());
+                {
+                    for (short i = 0; i < iFollowerSlots; ++i)
+                    {
+                        m_followers.emplace_back(pChar->GetUID());
```

Comment:
We could do that better, by making m_followers a vector of std::pair<ushort, CUID>, where the first number is the followerslots, value the second the UID.

Reply:
I can try, but I think Im not capable of doing that. Wouldnt that need more complex changes in the whole code?

Reply:
I suppose its defined as CUID already 
`std::vector<CUID> m_followers;` which means we would have to review every code where is it used?

Reply:
Im sorry im not able to do that, I guess if we change the def of the vector, we need to change every part where the FollowersUpdate is called/used, and even with the AI help, I couldnt make it work. 😥
Anyone ideas or more experience?

Reply:
I will take care of it, we'll need to change other code to make it working but it will be worth it.